src/set_theory/cardinal.lean

```diff
@@ -512,6 +512,13 @@ theorem lift_mk_le {α : Type u} {β : Type v} :
 ⟨λ ⟨f⟩, ⟨embedding.congr equiv.ulift equiv.ulift f⟩,
  λ ⟨f⟩, ⟨embedding.congr equiv.ulift.symm equiv.ulift.symm f⟩⟩
 
+-- A variant of `lift_mk_le` with universes specialized as `w = max u v`.
+-- This is sometimes necessary to avoid universe unification issues.
```

Comment:
Should this be a docstring?
```suggestion
/-- A variant of `lift_mk_le` with universes specialized as `w = max u v`.
This is sometimes necessary to avoid universe unification issues. -/
```